### ISSUE TITLE
Import error under windows

### ISSUE THREAD

**Issue by gjwo** [psycopg/psycopg2#329]
Error using psycopg2 under Anaconda Python 2.7, called from nilmtk

```
======================================================================
ERROR: Failure: ImportError (DLL load failed: %1 is not a valid Win32 application.)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\GJWood\Anaconda\lib\site-packages\nose\loader.py", line 420, in loadTestsFromName
    addr.filename, addr.module)
  File "c:\Users\GJWood\Anaconda\lib\site-packages\nose\importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "c:\Users\GJWood\Anaconda\lib\site-packages\nose\importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "c:\Users\GJWood\nilmtk\nilmtk\dataset_converters\__init__.py", line 6, in <module>
    from .dataport.download_dataport import download_dataport
  File "c:\Users\GJWood\nilmtk\nilmtk\dataset_converters\dataport\download_dataport.py", line 9, in <module>
    import psycopg2 as db
  File "c:\Users\GJWood\Anaconda\lib\site-packages\psycopg2\__init__.py", line 50, in <module>
    from psycopg2._psycopg import BINARY, NUMBER, STRING, DATETIME, ROWID
ImportError: DLL load failed: %1 is not a valid Win32 application.
```

**Comment by gjwo** [comment]
Environment issue fixed, now working

**Comment by rahul3103** [comment]
Hey I am also getting same error. Please tell me how do you solve this problem. I have also asked this question on Stack link: http://stackoverflow.com/questions/31934051/how-to-install-psycopg2-on-windows-7

**Comment by gjwo** [comment]
I had multiple issues with my python path and other environment variables, including more than one compiler, so some of the dependencies were in the environment of a different version of the compiler. when I sorted all of those out and did a fresh install of the package it worked.

**Comment by rahul3103** [comment]
Thanks for replying, I got it sorted. I was trying to install different architecture 64bit whereas my current Python version was 32 bit.

**Comment by rbrinck** [comment]
I'm still having issues with this:

`File "C:\Users\Username\AppData\Local\Programs\Python\Python35-32\lib\site-packages\psycopg2-2.6.1-py3.5-win32.egg\psycopg2\extensions.py", line 35, in <module>
		from psycopg2._psycopg import UNICODE, INTEGER, LONGINTEGER, BOOLEAN, FLOAT
	ImportError: DLL load failed: The specified module could not be found.`

I've tried putting everything I could think of in my PATH. I've tried different variations of "psycopg2._psycopg". I've downloaded everything I know to download (and made sure it was 64 bit like my computer). Any suggestions?
